db/people.yml

```diff
@@ -636,6 +731,48 @@ cyshi:
       group: bg
   name: Chenyang Shi
   position: Scientist
+dpodgorski:
+  active: false
+  aka: []
+  avatar: https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mp&f=y
+  bio: bio
+  education:
+    - begin_year: 0
+      institution: fix.me
+      degree: fix.me
+  employment:
+    - begin_year: 2012
+      begin_month: 11
+      position: Visiting Researcher
+      organization: columbiau
+      advisor: sbillinge
+      group: bg
+    - begin_year: 2012
+      begin_month: 3
+      position: Visiting Researcher
+      organization: columbiau
+      advisor: sbillinge
```

Comment:
Dragica was a student with Martin Schmidt who is somewhere in our contacts.yml

Reply:
I found when she finished her phd, but not her begin year :/

Reply:
just subtract 3 for a German PhD.